src/settings.env

```diff
@@ -0,0 +1,5 @@
+DEBUG=True
```

Comment:
This file must not be commit, regardless the file name should be .env and not settings.env, also moves this to the main dir and add to gitignore file, if you want you can include a .env.example file with the vars bane add describing which  should be the values